sunkit_magex/pfss/tests/test_tracers.py

```diff
@@ -9,9 +9,8 @@
 from sunkit_magex.pfss import tracing
 
 
-@pytest.fixture(params=[tracing.PythonTracer(),
-                        tracing.FortranTracer()],
-                ids=['python', 'compiled'])
+@pytest.fixture(params=[tracing.PerformanceTracer()],
+                ids=['compiled'])
```

Comment:
surely we don't need params here at all any more?

Reply:
yeah ill add back the python one.